packages/react-native-codegen/src/parsers/flow/index.js

```diff
@@ -13,19 +13,40 @@
 import type {SchemaType} from '../../CodegenSchema.js';
 
 const fs = require('fs');
-const {buildSchema} = require('./buildSchema');
+
+const {buildSchema} = require('../parsers-commons');
+const {Visitor} = require('./Visitor');
 const {FlowParser} = require('./parser');
+const {buildComponentSchema} = require('./components');
+const {wrapComponentSchema} = require('./components/schema');
+const {buildModuleSchema} = require('./modules');
 
 const parser = new FlowParser();
 
 function parseModuleFixture(filename: string): SchemaType {
   const contents = fs.readFileSync(filename, 'utf8');
 
-  return buildSchema(contents, 'path/NativeSampleTurboModule.js', parser);
+  return buildSchema(
+    contents,
+    'path/NativeSampleTurboModule.js',
+    wrapComponentSchema,
+    buildComponentSchema,
+    buildModuleSchema,
+    Visitor,
+    parser,
+  );
```

Comment:
can we call `parseString` here, instead of calling `buildSchema` directly?
In this way, if the `buildSchema` signature has to change, we only need to update one method instead of two.

Reply:
Oh yeah that's a really good idea. I will look at it

Reply:
Done ✅